theme/sass/components/_testimonials.scss

```diff
@@ -0,0 +1,30 @@
+/* Testimonials */
+
+.testimonial-container.swiper-container {
+    & .full-width {
```

Comment:
there shouldn't be a space between ampersand and .full-width

Reply:
done